project_report/project_report_latex.py

```diff
@@ -396,25 +401,17 @@ def create_results_section(self):
 
     def create_file_format_section(self):
         with self.doc.create(Section('Format of the Files Delivered', numbering=True)):
-            for format_delivered in self.pi.get_format_delivered():
-                if format_delivered == 'fastq':
-                    with self.doc.create(Subsection('Fastq format', numbering=True)):
-                        add_text(self.doc, report_text.get('fastq_format'))
-                        self.doc.append(NoEscape('\n'))
-                        self.doc.append('More detail about the format in ')
-                        self.doc.append(HRef(url=NoEscape(report_text.get('fastq_link')), text='Fastq specification'))
-                if format_delivered == 'bam':
-                    with self.doc.create(Subsection('BAM format', numbering=True)):
-                        add_text(self.doc, report_text.get('bam_format'))
-                        self.doc.append(NoEscape('\n'))
-                        self.doc.append('More detail about the format in ')
-                        self.doc.append(HRef(url=NoEscape(report_text.get('bam_link')), text='BAM specification'))
-                if format_delivered == 'vcf':
-                    with self.doc.create(Subsection('VCF format', numbering=True)):
-                        add_text(self.doc, report_text.get('vcf_format'))
+            formats_delivered = self.pi.get_format_delivered()
+            for fformat in ['fastq', 'bam', 'vcf']:
```

Comment:
This is down to personal preference, but I personally would prefer file_format, as fformat could look like a typo.

Reply:
Done